redirect.go

```diff
@@ -191,11 +205,34 @@ func (r *Redirect) To(location string) error {
 // If you want to send queries or params to route, you should use config parameter.
 func (r *Redirect) Route(name string, config ...RedirectConfig) error {
 	// Check config
-	cfg := RedirectConfig{}
+	cfg := RedirectConfig{CookieConfig: CookieConfigDefault}
+
 	if len(config) > 0 {
 		cfg = config[0]
 	}
 
+	if cfg.CookieConfig == (CookieConfig{}) {
+		cfg.CookieConfig = CookieConfigDefault
+	}
+
+	if cfg.CookieConfig.Name == "" {
+		cfg.CookieConfig.Name = CookieConfigDefault.Name
+	}
+
+	if cfg.CookieConfig.SameSite == "" {
+		cfg.CookieConfig.SameSite = CookieConfigDefault.SameSite
+	}
+
+	// RFC6265: If SameSite=None, the Secure attribute must be set
+	// https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-14#name-storage-model
+	// Section 5.6.19
+	if utils.ToLower(cfg.CookieConfig.SameSite) == "none" {
+		cfg.CookieConfig.Secure = true // Ensure Secure is true if SameSite=None
+	}
+
+	// Set the cookie configuration in the Redirect struct
+	r.cookieConfig = cfg.CookieConfig
```

Comment:
since the part with the configurations has become larger, we can move this to a private function for creating the configuration, so that it becomes more usable again